### DISCUSSION TITLE
AppKit for Cosmos

### DISCUSSION THREAD

**Discussion by aledecicco** [WalletConnect/walletconnect-monorepo#5302] [Developer Support]
I'm looking to integrate AppKit into my React web app. I need support for Cosmos chains, but the docs don't say anything about it. I found [this issue](https://github.com/WalletConnect/web3modal/issues/709) in your repo related to that, and it was closed as completed some time ago, so I figured maybe this feature is available somewhere. 

I want to use AppKit instead of Walletconnectmodal because I need support for browser extensions (and social logins as well would be a plus). Also, I wouldn't mind using a beta version of the library if that gave me the features I need, but I can't find that either.

Thank you!

**Comment by o-az** [comment]
I'm curious too.
